orc/orcarm.c

```diff
@@ -1300,6 +1300,71 @@ orc_arm64_emit_lg (OrcCompiler *p, OrcArm64RegBits bits, OrcArm64DP opcode,
   orc_arm_emit (p, code);
 }
 
+/** data processing instructions: move wide
+ *
+ * General formats
+ *    3                   2                   1
+ *  1 0 9 8 7 6 5 4 3 2 1 0 9 8 7 6 5 4 3 2 1 0 9 8 7 6 5 4 3 2 1 0
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |b|op |1 0 0 1 0 1|hw |              imm16            |    Rd   |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ */
+
+#define arm64_code_mov_wide(b,op,hw,imm,Rd) (0x12800000 | ((((b)==64)&0x1)<<31) | \
```

Comment:
What do you intend to do with ```(((b)==64)&0x1)``` ?

If you want 1 for true and 0 for false, use ```!!((b) == 64)```

Reply:
Yes, I just want to keep the same format like other variables.
your suggestion looks better.